bin/sandbox

```diff
@@ -39,6 +39,11 @@ fi
 
 cd ./sandbox
 
+# Add support for tracking coverage
+unbundled bundle add simplecov -v 0.22 --require=false --skip-install
+unbundled bundle add codecov --require=false
+echo "require_relative '../../coverage.rb' if ENV['COVERAGE']" >> config/boot.rb
+
```

Comment:
I'm confused. Shouldn't it go into the dummy-app? :thinking:

Reply:
The starter was sort of the pioneer in this approach and does everything with the sandbox, eventually we'll want to normalize things across repositories and extract some tooling I guess, but for now that's the setup 😒

Reply:
Oh, I see. Thanks for clarifying.